### PR TITLE
fix: Log out user menu on Django 5.0

### LINKED ISSUE #509: "Django administration > Logout user" leads to a 405 error (Method Not Allowed (GET): /en/admin/logout/)
### Summary
If I go to **_http://localhost:8000/en/admin/cms/pagecontent/_** and then click **_Django administration > Logout user_**,  a 405 error is raised, and "Method Not Allowed (GET): /en/admin/logout/" is printed on the console.
Screenshot from 2024-01-19 14-37-34

But if I go to the preview mode of a page (e.g. http://localhost:8000/en/admin/cms/placeholder/object/10/preview/4/), **_Django administration_** link is replaced by **_example.com_** and then **_example.com > Logout user_** works like a charm.

Screenshot from 2024-01-19 14-38-29

### Expected behavior
**_Django administration > Logout user_** should work normally like **_example.com > Logout user_**.

### Actual behavior
**_Django administration > Logout user_** raises a 405 error.

### Environment
* Python version: 3.11.7
* Django version: 5.0.1
* django CMS version: 4.1.0

### PR DESCRIPTION
**Pull request by erkesado**
Fixes #509

### CODE REVIEW

**Review by fsbraun — APPROVED** [review]
Nice. Just tested and looks perfect. Good job!

File: djangocms_admin_style/sass/components/_header.scss [hunk 1]
```diff
@@ -131,6 +131,41 @@
                             background: $color-primary;
                         }
                     }
+                    form {
+                        box-shadow: none;
+                        margin: 0;
+                        padding: 0;
+                        button {
```
[the remark this answers is not in the document]
Comment (erkesado) [reply djangocms_admin_style/sass/components/_header.scss:138]: @fsbraun sure

**Comment by erkesado** [comment]
@fsbraun I'm glad that it works :D

File: djangocms_admin_style/sass/components/_header.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (erkesado) [reply djangocms_admin_style/sass/components/_header.scss]: @fsbraun I've removed the redundant code and committed it.

**Comment by fsbraun** [comment]
@erkesado Frankly, I was hoping for more common styles between `<a>` and `<button>`. Now, that you've done, it seems less transparent to me. Sorry, I was pointing you in the wrong direction. 
Can you go back one commit to  `c49ddcb96c7438861c94c58634f7b1aa252578a7` (`git reset HEAD~1 && git push --force`, I believe)?

**Comment by erkesado** [comment]
@fsbraun I restored the initial commit

**Pull request merged**